cartridge-cli.lua

```diff
@@ -1270,64 +1269,89 @@ RUN groupadd -r tarantool \
     && chown tarantool:tarantool /var/lib/tarantool \
     && mkdir -p /var/run/tarantool/ --mode 755 \
     && chown tarantool:tarantool /var/run/tarantool
-
-${install_tarantool}
-
-RUN echo 'd /var/run/tarantool 644 tarantool tarantool' > /usr/lib/tmpfiles.d/${name}.conf \
-    && chmod 644 /usr/lib/tmpfiles.d/${name}.conf
-
-# copy application source code
-COPY . ${dir}
-
-WORKDIR ${dir}
-
-RUN : "----------- pre-build -----------" \
-    && \
-    if [ -f ${prebuild_script_name} ]; then \
-       set -e && . ${prebuild_script_name} \
-       && rm ${prebuild_script_name}; \
-    fi \
-    && \
-    : "------------- build -------------" \
-    && \
-    if ls *.rockspec 1> /dev/null 2>&1; then \
-        tarantoolctl rocks make; \
-    fi \
-    && \
-    : "----------- post-build -----------" \
-    && \
-    if [ -f ${postbuild_script_name} ]; then \
-        set -e && . ${postbuild_script_name} \
-        && rm ${postbuild_script_name}; \
-    fi
-
-USER tarantool:tarantool
-
-CMD TARANTOOL_WORKDIR=${workdir}.${instance_name} \
-    TARANTOOL_PID_FILE=/var/run/tarantool/${name}.${instance_name}.pid \
-    TARANTOOL_CONSOLE_SOCK=/var/run/tarantool/${name}.${instance_name}.control \
-    tarantool ${dir}/init.lua
 ]]
 
 local DOCKER_INSTALL_OPENSOURCE_TARANTOOL_TEMPLATE = [[
-# install opensource Tarantool
+### Install opensource Tarantool
 RUN curl -s \
         https://packagecloud.io/install/repositories/tarantool/${tarantool_repo_version}/script.rpm.sh | bash \
```

Comment:
Hmm... But in case 2.1 tarantool_repo_version = 2x (https://packagecloud.io/install/repositories/tarantool/2x/script.rpm.sh)

Reply:
OMG, thanks!

I will fix it in a separated patch.

Reply:
See #146